### ISSUE TITLE
[Feature Request] Add a "Save as Main Configuration" option in the "Configuration File" menu

### ISSUE THREAD

**Issue by bslenul** [libretro/RetroArch#13222]
Hey!

I think it'd be really nice if we could save a loaded config file as the main config file, currently the loaded config file will only be active for the current session. It would be really useful when starting fresh to load your own defaults for example. Like I have a file I've created after a fresh install of RA, with a few settings changed that I like as my defaults, if I want to load them on a fresh install I have to manually move and rename the config file. Would be super useful for devices where you can't easily manipulate `retroarch.cfg` as well, like in later Android versions I believe the file is now in a root only folder.

It would be nice if we could do something like that:

1. Start a fresh install of RetroArch.
2. Go to `Main Menu > Configuration File > Load Configuration` and load your config file with your defaults.
3. Click `Save as Main Configuration`, and the changes would be saved in `retroarch.cfg` directly, instead of being temporary.

**Comment by hizzlekizzle** [comment]
Does that not happen if you just load it up and then quit RetroArch? Or will it just write back to the one you loaded? I was under the impression it always wrote to retroarch.cfg on exit.

**Comment by bslenul** [comment]
Nope, changes made while an "external" config is loaded are saved to that specific config file on exit, not `retroarch.cfg`.

**Comment by Sanaki** [comment]
This is of particular importance on Android, where users have no direct access to the default config location.
